src/test/java/org/sasanlabs/service/vulnerability/commandInjection/CommandInjectionTest.java

```diff
@@ -0,0 +1,592 @@
+            throws ServiceApplicationException, IOException {
+        RequestEntity entity = new RequestEntity(HttpMethod.GET, URI.create(Strings.EMPTY));
+        ResponseEntity<GenericVulnerabilityResponseBean<String>> responseEntity =
+                commandInjection.getVulnerablePayloadLevel2(Strings.EMPTY, entity);
+
+        // Assert
+        assertStatusAndValidate(responseEntity, Strings.EMPTY, Strings.EMPTY);
+
+        verify(commandInjection).getVulnerablePayloadLevel2(eq(Strings.EMPTY), eq(entity));
+    }
+
+    @Test
+    public void getVulnerablePayloadLevel2_LocalhostAndSEMICOLON_ExpectNoResponseContent()
+            throws ServiceApplicationException, IOException {
+        RequestEntity entity =
+                new RequestEntity(
+                        HttpMethod.GET,
+                        URI.create(
+                                "http://localhost:9090/CommandInjection/LEVEL_2?ipaddress=localhost;"));
+        ResponseEntity<GenericVulnerabilityResponseBean<String>> responseEntity =
+                commandInjection.getVulnerablePayloadLevel2(LOCALHOST, entity);
+
+        // Assert
+        assertStatusAndValidate(responseEntity, LOCALHOST, Strings.EMPTY);
+
+        verify(commandInjection).getVulnerablePayloadLevel2(eq(LOCALHOST), eq(entity));
+    }
+
+    @Test
+    public void getVulnerablePayloadLevel2_LocalhostPlusAnd_ExpectNoResponseContent()
+            throws ServiceApplicationException, IOException {
+        RequestEntity entity =
+                new RequestEntity(
+                        HttpMethod.GET,
+                        URI.create(
+                                "http://localhost:9090/CommandInjection/LEVEL_2?ipaddress=localhost&"));
+        ResponseEntity<GenericVulnerabilityResponseBean<String>> responseEntity =
+                commandInjection.getVulnerablePayloadLevel2(LOCALHOST, entity);
+
+        // Assert
+        assertStatusAndValidate(responseEntity, LOCALHOST, Strings.EMPTY);
+
+        verify(commandInjection).getVulnerablePayloadLevel2(eq(LOCALHOST), eq(entity));
+    }
+
+    @Test
+    public void getVulnerablePayloadLevel2_LocalhostAnd20_ExpectPingSuccess()
+            throws ServiceApplicationException, IOException {
+        RequestEntity entity =
+                new RequestEntity(
+                        HttpMethod.GET,
+                        URI.create(
+                                "http://localhost:9090/CommandInjection/LEVEL_2?ipaddress=localhost%20%7c%20cat%20/etc/passwd"));
+        ResponseEntity<GenericVulnerabilityResponseBean<String>> responseEntity =
+                commandInjection.getVulnerablePayloadLevel2(LOCALHOST, entity);
+
+        // Assert
+        assertStatusAndValidate(responseEntity, LOCALHOST, mockedSuccessPingResponse);
+
+        verify(commandInjection).getVulnerablePayloadLevel2(eq(LOCALHOST), eq(entity));
+    }
+
+    @Test
+    public void getVulnerablePayloadLevel2_LocalhostAnd26_ExpectPingSuccess()
```

Comment:
can be named `localhostAndUrlEncoded&`  or  `localhostAndUrlEncodedand` instead of `LocalhostAnd26`